src/ontology/cl-edit.owl

```diff
@@ -31919,6 +31920,17 @@ AnnotationAssertion(rdfs:label obo:CL_4040005 "mesenchymal stem cell of apical p
 SubClassOf(obo:CL_4040005 obo:CL_0000134)
 SubClassOf(obo:CL_4040005 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003677))
 
+# Class: obo:CL_4042001 (TAC3-positive medium spiny neuron)
+
```

Comment:
Could you please add a date, using the annotation dcterms:date?

Reply:
This should be automated as part of the add new terms settings configured in Protege.  Ditto addition of ORCID. Latest details should be in Obook.